liquibase-core/src/main/java/liquibase/snapshot/jvm/JdbcSnapshotGenerator.java

```diff
@@ -103,9 +103,6 @@ public void addStatusListener(DiffStatusListener listener) {
     }
 
     protected void updateListeners(String message) {
-        if (this.statusListeners == null) {
```

Comment:
@arturobernalg or @MalloD12, hi! Why is this check for null statusListeners no longer needed? In other places where `this.something == null` the code was updated to `this.something.isEmpty()`. I'm just curious why we didn't need to do that here.

Reply:
Hi - it's because the variable has been initialized at line 27 and now it is final, so nothing can change it's value to null.

Reply:
Ah! Makes perfect sense. Thank you!